openhtf/plugs/__init__.py

```diff
@@ -117,7 +118,7 @@ def TearDown(self):
     pass
 
 
-def requires(update_kwargs=True, **plugs):
+def plug(update_kwargs=True, **plugs):
```

Comment:
bah, yeah i figured i'd have missed a couple things, but wanted to get it sent out before i went to sleep, will go back and fix all these types of things today.